ResearchKit/Common/ORKHelpers.h

```diff
@@ -162,6 +162,7 @@ UIColor *ORKRGB(uint32_t x);
 UIColor *ORKRGBA(uint32_t x, CGFloat alpha);
 
 id ORKFindInArrayByStudyId(NSArray *array, NSString *studyIdentifier);
+id ORKFindInArrayByFormItemId(NSArray *array, NSString *formItemIdentifier);
```

Comment:
If this utility is only used within `ORKRegistrationStep`, move it into ORKRegistrationStep.m?
And `ORKFindInArrayByStudyId` is not being used in anywhere, consider to drop it too?